dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/SiteViewStrategy.java

```diff
@@ -0,0 +1,47 @@
+package com.dotmarketing.portlets.contentlet.transform.strategy;
+
+import com.dotcms.api.APIProvider;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.liferay.portal.model.User;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Strategy for Site View
+ * Convert an object to Host in order to hydrated
+ * @author jsanca
+ */
+public class SiteViewStrategy extends AbstractTransformStrategy<Contentlet> {
+
+    /**
+     * Main Constructor
+     * @param toolBox
+     */
+    SiteViewStrategy(final APIProvider toolBox) {
+        super(toolBox);
+    }
+
+    /**
+     * transform method entry point
+     * @param contentlet
+     * @param map
+     * @param options
+     * @param user
+     * @return
+     * @throws DotDataException
+     */
+    @Override
+    protected Map<String, Object> transform(final Contentlet contentlet,
+                                            final Map<String, Object> map, final Set<TransformOptions> options, final User user) throws DotDataException {
+
+        if (Host.HOST_VELOCITY_VAR_NAME.equals(contentlet.getContentType().variable())) {
+
+            final Host host = new Host(contentlet);
+            map.putAll(host.getMap());
+        }
+        return map;
```

Comment:
not sure if it's gonna be needed but we could also have here an options to render the site as an exploded field 
@Override
    protected Map<String, Object> transform(final Contentlet contentlet,
                                            final Map<String, Object> map, final Set<TransformOptions> options, final User user) throws DotDataException {
```
        if (Host.HOST_VELOCITY_VAR_NAME.equals(contentlet.getContentType().variable())) {
    
        } if(options.contains(SITE_AS_MAP)){
            final Host host = new Host(contentlet);
            map.put("hostId",host.getHostIdentifier());
            map.put("hostName",host.getHostName());
    
```